lib/ex_aws/util/s3_object.ex

```diff
@@ -0,0 +1,3 @@
+defmodule S3Object do
```

Comment:
Shouldn't this be `ExAws.Rekognition.S3Object` ? This is only used in this project, right ?

Reply:
Yes, true 👍 But I am still not sure if Ben wants to use this object on the rest of the libraries, idealy it should be inside the base ex_aws library or the util/model library